src/Evolution/Systems/NewtonianEuler/Limiters/Weno.cpp

```diff
@@ -55,7 +55,8 @@ bool characteristic_simple_weno_impl(
         std::pair<Direction<VolumeDim>, ElementId<VolumeDim>>,
         typename NewtonianEuler::Limiters::Weno<VolumeDim>::PackagedData,
         boost::hash<std::pair<Direction<VolumeDim>, ElementId<VolumeDim>>>>&
-        neighbor_data) noexcept {
+        neighbor_data,
+    const bool compute_char_transformation_numerically) noexcept {
```

Comment:
formatting looks suspicious

Reply:
This is what `clang-format` produces, and I agree with it: the lines above are a single very big multi-line argument, so the additional lines for that argument get indented a little further.